engine/orchestration/src/main/java/com/cloud/agent/manager/AgentManagerImpl.java

```diff
@@ -1399,7 +1399,7 @@ private void processHostHealthCheckResult(Boolean hostHealthCheckResult, long ho
             }
             if (!BooleanUtils.toBoolean(EnableKVMAutoEnableDisable.valueIn(host.getClusterId()))) {
                 logger.debug("{} is disabled for the cluster {}, cannot process the health check result " +
-                        "received for the host {}", EnableKVMAutoEnableDisable.key(), host.getClusterId(), host);
+                        "received for the host {}", EnableKVMAutoEnableDisable.key(), host.getClusterId(), host.getName());
```

Comment:
I would prefer this without `host.getName()` change as then it would give more details of hosts using 
```suggestion
                        "received for the {}", EnableKVMAutoEnableDisable.key(), host.getClusterId(), host);
```